src/mainviews/job-detail.tsx

```diff
@@ -109,203 +94,172 @@ export function JobDetail(props: IJobDetailProps): JSX.Element {
+        <FormLabel sx={{ mb: 4 }} component="legend">
+          {trans.__('Parameters')}
+        </FormLabel>
+        <Grid container spacing={4}>
+          {Object.entries(job?.parameters ?? {}).map(([parameter, value]) => (
+            <React.Fragment key={parameter}>
+              <Grid xs={12} md={6}>
                 <TextFieldStyled
-                  label={trans.__('Start time')}
-                  defaultValue={timestampLocalize(job?.start_time ?? '')}
+                  label={trans.__('Parameter')}
+                  defaultValue={parameter}
                 />
+              </Grid>
+              <Grid xs={12} md={6}>
                 <TextFieldStyled
-                  label={trans.__('End time')}
-                  defaultValue={timestampLocalize(job?.end_time ?? '')}
+                  label={trans.__('Value')}
+                  defaultValue={value}
                 />
-              </Stack>
-            </AccordionDetails>
-          </Accordion>
-
-          <Accordion defaultExpanded={true}>
-            <AccordionSummary
-              expandIcon={<caretDownIcon.react />}
-              aria-controls="panel-content"
-              id="panel-header"
-            >
-              <FormLabel component="legend">
-                {trans.__('Advanced options')}
-              </FormLabel>
-            </AccordionSummary>
-            <AccordionDetails id="panel-content">
-              <Stack component="form" spacing={4}>
-                Placeholder
-              </Stack>
-            </AccordionDetails>
-          </Accordion>
-        </>
-      );
-    }
-  }
+              </Grid>
+            </React.Fragment>
+          ))}
+        </Grid>
+      </CardContent>
+    </Card>
+  );
 
-  return (
-    <>
-      <Box sx={{ maxWidth: '500px', p: 4 }}>
+  const OtherOptions = () => (
+    <Card>
+      <CardContent>
         <Stack spacing={4}>
-          <div role="presentation">
-            <Breadcrumbs aria-label="breadcrumb">
-              <Link
-                underline="hover"
-                color="inherit"
-                onClick={(
-                  _:
-                    | React.MouseEvent<HTMLAnchorElement, MouseEvent>
-                    | React.MouseEvent<HTMLSpanElement, MouseEvent>
-                ): void => props.setView('ListJobs')}
-              >
-                {trans.__('Notebook Jobs')}
-              </Link>
-              <Typography color="text.primary">{props.model.jobId}</Typography>
-            </Breadcrumbs>
-          </div>
-          <Heading level={1}>{trans.__('Job Detail')}</Heading>
-          <MainArea />
+          <TextFieldStyled
+            label={trans.__('Idempotency token')}
+            defaultValue={job?.idempotency_token ?? ''}
+          />
+          <FormLabel component="legend">{trans.__('Tags')}</FormLabel>
+          {job?.tags &&
+            job?.tags.map(tag => (
+              <TextFieldStyled label={trans.__('Tag')} defaultValue={tag} />
```

Comment:
See #56 — the label `Tag` for every tag is redundant. Perhaps Iterate on `(tag, idx)` and use `idx + 1` (index plus one) in the tag name: `trans.__('Tag %1', idx + 1)`. This will make each successive tag have a distinct title.

Reply:
That's a good suggestion for readability, implemented